_posts/EternalFootman/2016-01-14-turtles.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+author: EternalFootman
+title: "Kit's Turtles"
+---
+
+Here are Tina's desert adventures.
+
+<iframe src="https://trinket.io/embed/python/6e3d7d7d02?start=result" width="100%" height="356" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+And here is a game of turtle checkers.
+
+<iframe src="
+https://trinket.io/embed/python/582996e6a8?start=result" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+This one involved a lot of repetition, interspersed with turning back and forth to make sure everything faced in the correct direction.
+Below is the code that made one row of pieces (there may be a far more efficient method, but this worked eventually).
+
+```
+tina.stamp()
+tina.left(90)
+tina.forward(100)
+tina.right(90)
+tina.stamp()
+tina.left(90)
+tina.forward(100)
+tina.right(90)
+tina.stamp()
+tina.left(90)
+tina.forward(100)
+tina.right(90)
+tina.stamp()
+```
+
+I had some difficulty making a proper board; ideally, the squares will alternate in color. Any thoughts on how to accomplish this?
```

Comment:
Good question!  I'd basically make the squares you wanted to be white with some fill after the main board was drawn.  If the lines were black but you turned white before each `fill(False`) I think you'd get the effect you wanted.

What you're trying to do will be *so* much easier once we learn functions.  Feel free to read ahead if you want.  I'd make a function `square(x, y, color)` that took integers x and y and a color name and made a square at that location.